python/cudf/cudf/core/indexing.py

```diff
@@ -4,12 +4,16 @@
 
 import numpy as np
 import pandas as pd
+from cupy import isscalar
```

Comment:
I think we might want to use [our own is_scalar](https://github.com/rapidsai/cudf/blob/branch-0.19/python/cudf/cudf/utils/dtypes.py#L302-L317) where we can for consistency